contracts/MainChain/TransferUnlock.es

```diff
@@ -43,8 +43,7 @@
         selfOutput.propositionBytes == SELF.propositionBytes &&
         selfOutput.tokens == SELF.tokens &&
         selfOutput.R4[Coll[Byte]].get == sideChainStateHash &&
-        outHeight <= HEIGHT &&
-        outHeight >= HEIGHT - 5
+        outHeight >= HEIGHT + 5
```

Comment:
I might be misunderstanding but I thought `outHeight` is meant to require 5 block confs to complete the unlock process, don't think original code was doing this

Reply:
It is not about 5 blocks to unlock, rather, to fix current HEIGHT. As strict equality would break validation on the way, we have 5 blocks buffer for inclusion into blockchain ( then tx would become invalid)